easybuild/easyconfigs/g/GTS/GTS-0.7.6-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'GTS'
+version = '0.7.6'
+
+homepage = 'http://gts.sourceforge.net/'
+description = """GTS stands for the GNU Triangulated Surface Library.
+ It is an Open Source Free Software Library intended to provide a set of useful
+ functions to deal with 3D surfaces meshed with interconnected triangles."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['059c3e13e3e3b796d775ec9f96abdce8f2b3b5144df8514eda0cc12e13e8b81e']
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
```

Comment:
@fizwit `binutils` *must* be included as a build dependency whenever `GCCcore` is used as a toolchain, so please include `('binutils', '2.30')` here as well

Reply:
('binutils', '2.30') added to GTS